xrdp/xrdp_wm.c

```diff
@@ -567,7 +567,10 @@ xrdp_wm_init(struct xrdp_wm *self)
     xrdp_wm_load_static_pointers(self);
     self->screen->bg_color = self->xrdp_config->cfg_globals.ls_top_window_bg_color;
 
-    if (self->session->client_info->rdp_autologin || self->hide_log_window)
+    /* fix black screen if hidelogwindow=yes in xrdp.ini, only the log window should be hidden. not everything */
+    /* https://github.com/neutrinolabs/xrdp/issues/414 */
+    /*if (self->session->client_info->rdp_autologin || self->hide_log_window)*/
```

Comment:
The fix seems good but don't leave the old code in comment. You can just remove it.

Reply:
should I remove all other comments or leave them in place?

Reply:
I think no comments needed there.